src/main/java/uk/gov/hmcts/reform/jps/domain/JohPayroll.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.jps.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
+
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Builder
+@NoArgsConstructor()
+@AllArgsConstructor
+@Data
```

Comment:
Please replace @Data with @Setter and @Getter as this causes problems during fetching records

Reply:
done